src/System.Data.SqlClient/src/Resources/Strings.resx

```diff
@@ -1180,7 +1180,4 @@
   <data name="MDF_UnableToBuildCollection" xml:space="preserve">
     <value>Unable to build schema collection '{0}';</value>
   </data>
-  <data name="AmbientTransactionsNotSupported" xml:space="preserve">
```

Comment:
You'll also need to remove the exceptions that reference this. Build the project, and they'll show up in the build errors.